### ISSUE TITLE
Confusing wording in the contributors guide

### ISSUE THREAD

**Issue by justinGilmer** [glotzerlab/signac-flow#105]
In the [contributor's guide](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md#general-guidelines), it says that all PRs for features or changes should be made against `master`. However, the [template](https://github.com/glotzerlab/signac-flow/blob/master/.github/PULL_REQUEST_TEMPLATE.md#L17-L19) that is provided to the user when submitting their pull request mentions that they should be using either `master` or `develop` depending on the PR, which can lead to confusion

Here is the snippet from the PR template
```markdown
This pull request is based on
- [ ] *master*, because it is a bug fix or an update to the documentation.
- [ ] *develop*, because it introduces a new feature.
```
```markdown
* Use the [OneFlow](https://www.endoflineblog.com/oneflow-a-git-branching-model-and-workflow) model of development:
    - Both new features and bug fixes should be developed in branches based on `master`.
- Hotfixes (critical bugs that need to be released *fast*) should be developed in a branch based on the latest tagged release.
```

I am not sure what the correct option is, just bringing it up.

**Comment by csadorf** [comment]
Thx for reporting! I thought we had updated it, but apparently dropped the ball somewhere.

**Comment by csadorf** [comment]
Fixed with merge of PR #106 .

**Comment by csadorf** [comment]
Fixed with merge of PR #106 .
